src/btreemap/proptests.rs

```diff
@@ -196,5 +200,33 @@ fn execute_operation<M: Memory>(
                 assert_eq!(btree.remove(&k), Some(v));
             }
         }
+        Operation::PopLast => {
+            assert_eq!(std_btree.len(), btree.len() as usize);
+            if std_btree.is_empty() {
+                return;
+            }
+
+            let idx = std_btree.len();
+
+            if let Some(entry) = btree.iter().skip(idx).take(1).next() {
+                eprintln!("Pop last, the key is {}", hex::encode(&entry.0));
```

Comment:
These printlns are written in a special format to be easy to replay a failed test case later.

```suggestion
                eprintln!("PopLast");
```